angularjs-1.5-basics/src/app/interval-example/_interval-example.js

```diff
@@ -0,0 +1,3 @@
+'use strict';
```

Comment:
__[jshint]__
* Use the function form of "use strict".

<!-- Comment by [SideCI](https://sideci.com) -->